frontend/src/metabase/redux/requests.js

```diff
@@ -0,0 +1,18 @@
+import { handleActions, createAction } from "metabase/lib/redux";
+import i from "icepick";
+
+const SET_REQUEST = "metabase/requests/SET_REQUEST";
+
+export const setRequest = createAction(SET_REQUEST, undefined, (payload, meta) => meta);
+
+export default handleActions({
+    //TODO: add a CLEAR_REQUEST action?
+    [SET_REQUEST]: {
+        next: (state, { payload }) => payload.id ?
```

Comment:
Good catch. Probably no longer matters when we refactor to use path arrays though.